plugins/notifications-backend/src/service/router.ts

```diff
@@ -250,27 +258,75 @@ export async function createRouter(
     res.send(status);
   });
 
-  router.post('/update', async (req, res) => {
+  const updateNotifications = async (
+    user: string,
+    ids: string[],
+    read?: boolean,
+    saved?: boolean,
+  ) => {
+    if (read === true) {
+      await store.markRead({ user, ids });
+    } else if (read === false) {
+      await store.markUnread({ user, ids });
+    }
+
+    if (saved === true) {
+      await store.markSaved({ user: user, ids });
+    } else if (saved === false) {
+      await store.markUnsaved({ user: user, ids });
+    }
+  };
+
+  router.put('/notification/:id', async (req, res) => {
```

Comment:
Wanna push back a lil bit on this, do we really need a separate endpoint for updating individual notifications when we have the the batch one?

If we wanna keep it I'd prolly make it a `PATCH` as discussed below

Reply:
Yeah, I think it's not needed, cleaning up